lib/gvl-tracing.rb

```diff
@@ -37,6 +37,11 @@ class << self
     def start(file)
       _start(file)
       @path = file
+
+      if block_given?
+        yield
```

Comment:
@ivoanjo we might wan to think about error handling 😄

Reply:
Done [53a21ca](https://github.com/ivoanjo/gvl-tracing/pull/14/commits/53a21ca9c3fdabd297ae19fb0fcb3237b10ac76e)